frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -195,10 +175,64 @@ class ChartSettings extends Component {
     ).some(widget => !widget.hidden);
   }
 
+  getStyleWidget = () => {
+    const widgets = this._getWidgets();
+    const series = this._getTransformedSeries();
+    const { currentWidget } = this.state;
+    const seriesSettingsWidget =
+      currentWidget && widgets.find(w => w.id === "series_settings");
+
+    if (seriesSettingsWidget) {
+      if (currentWidget.props?.seriesKey) {
+        return {
+          ...seriesSettingsWidget,
+          props: {
+            ...seriesSettingsWidget.props,
+            initialKey: currentWidget.props.seriesKey,
+          },
+        };
+      } else if (currentWidget.props?.initialKey) {
+        const settings = this._getSettings();
+        const singleSeriesForColumn = series.find(
+          single =>
+            getColumnKey(single.data.cols[1]) ===
+            currentWidget.props.initialKey,
+        );
+
+        const isBreakout = settings["graph.dimensions"]?.length > 1;
```

Comment:
I believe we have a bunch of checks in place for `graph.dimensions` length to determine if a chart has breakouts — possible opportunity to extract a simple predicate like `hasBreakouts`